Source/ThirdParty/bitmap-sdf.js

```diff
@@ -0,0 +1,143 @@
+define([], function() {
+
```

Comment:
I remember you mentioning this file was derived from `tiny-sdf`. It's worth adding a comment explaining that here.

Reply:
Yup it's a fork, pushed a change for this.